project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.block.inc

```diff
@@ -226,18 +226,18 @@ function _c4m_user_profile_block_user_my_profile_groups() {
     'groups' => $rendered_groups,
   );
 
-  $block['content'] = empty($rendered_groups) ? theme('c4m_user_profile_my_profile_groups', $variables) : FALSE;
+  $block['content'] = !empty($rendered_groups) ? theme('c4m_user_profile_my_profile_groups', $variables) : FALSE;
```

Comment:
please change this one in here:
https://github.com/capacity4dev/capacity4more/pull/1155/files#diff-3f72bf9fc8c0ee2206d46d044126106fR229